### ISSUE TITLE
Incorporate a section for comments in custom tickets with the plugin 

### ISSUE THREAD

**Issue by Ramadyfm** [3drepo/3drepo.io#4541]
### Description
Comments section does not exist when using the plugin for custom tickets, so currently there is no way to keep track of custom ticket history for customers who raising tickets in BIM authoring tools.

### Related Resources
Image showing no comments section in plugin for custom tickets 
![image](https://github.com/3drepo/3drepo.io/assets/100203626/33963d28-1f70-44c5-bfb1-5d4f1b2ff79a)

Comments section included in template in .io 

![image](https://github.com/3drepo/3drepo.io/assets/100203626/36adf4dd-67a2-4cb9-a89c-7cd6d8d7ace1)

**Comment by carmenfan** [comment]
yes this is a known, we don't have feature parity between the plugins and the web as the plugins implemented the PoC version of custom tickets (and web has since moved on.)

Will move this to "to discuss" on the roadmap so we can talk about the importance next roadmap meeting.
